src/store/Entity/Properties.ts

```diff
@@ -0,0 +1,9 @@
+import Dictionary from '@/common/interfaces/Dictionary';
```

Comment:
`Property` is a pretty loaded term in the Wikibase world. The convention for the state interface name that I've seen is `[ModuleName]State`, so this should be `EntityState.ts`